ui/appui-react/src/appui-react/imodel-components/category-tree/CategoryVisibilityHandler.tsx

```diff
@@ -17,15 +17,15 @@ const EMPTY_CATEGORIES_ARRAY: Category[] = [];
 
 /**
  * Loads categories from viewport or uses provided list of categories.
- * @internal
+ * @alpha
  */
 export function useCategories(viewManager: ViewManager, imodel: IModelConnection, view?: Viewport) {
```

Comment:
Are this and `loadCategoriesFromViewport` also used outside this package?

Reply:
i'll change `useCategories` and `loadCategoriesFromViewport` back to internal.

Bill, were you suggesting i leave everything else in this file marked as internal? I think we should be consistent what we do here with whats already done in the `ModelsVisibilityHandler`. I have no problem marking everything in there as internal as well.

Reply:
My 2 cents, is that we should make it all "Visibility" types that you need in vcr `internal` (that are alpha). Then wait till we have a need/request from external customers to officially make them public.

Reply:
I think the fact that they're consumed within VCR makes it clear that they are needed outside of the core repository to build something larger on top of the current APIs.

My take on `internal` however is probably controversial but I think it should be used for the majority of cases in two scenarios 1) as a way to export from one module to be used in another module within the same package and 2) used across the RPC boundaries which exist between the frontend/common/backend packages of some API as it is more or less an internal contract.

All other uses of it I find as a hindrance to the usage of our APIs, because if we need it to build a higher level component in some other package wouldn't someone else need it to do the same?

Reply:
I want to get this pr through before noon, one last vote? 2-1 alpha wins at the moment. Are we ok with that? 
@bsteinbk @saskliutas @calebmshafer

Reply:
Ended up going with alpha, resolving this thread.